eng/targets/Common.targets

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <Import Project="Packaging.targets" />
```

Comment:
Why is this line necessary?

Reply:
@jaredpar dunno, this was originally included in [Directory.Build.targets](https://github.com/dotnet/razor/blob/eef0189b5fbef8ad4446dc0e409163448dff81e9/Directory.Build.targets#L24C3-L24C53)

I refactored it into `Common.targets` and kept including it, there.